dist/colors.color-map.scss

```diff
@@ -1,90 +1,79 @@
 $polaris-colors: (
   'purple':
     (
-      'lighter': rgb(246, 240, 253),
-      'light': rgb(227, 208, 255),
-      'base': rgb(156, 106, 222),
-      'dark': rgb(80, 36, 143),
+      'text': rgb(80, 73, 90),
       'darker': rgb(35, 0, 81),
-      'text': rgb(80, 73, 90)
+      'dark': rgb(80, 36, 143),
+      'purple': rgb(156, 106, 222),
+      'light': rgb(227, 208, 255),
+      'lighter': rgb(245, 240, 252)
     ),
   'indigo':
     (
-      'lighter': rgb(244, 245, 250),
-      'light': rgb(179, 188, 245),
-      'base': rgb(92, 106, 196),
-      'dark': rgb(32, 46, 120),
+      'text': rgb(62, 65, 85),
       'darker': rgb(0, 6, 57),
-      'text': rgb(62, 65, 85)
+      'dark': rgb(32, 46, 120),
+      'indigo': rgb(92, 106, 196),
+      'light': rgb(179, 188, 245),
+      'lighter': rgb(243, 244, 250)
     ),
   'blue':
     (
-      'lighter': rgb(235, 245, 250),
-      'light': rgb(180, 225, 250),
-      'base': rgb(0, 122, 206),
+      'text': rgb(62, 78, 87),
+      'darker': rgb(0, 21, 42),
       'dark': rgb(8, 78, 138),
-      'darker': rgb(0, 20, 41),
-      'text': rgb(62, 78, 87)
+      'blue': rgb(0, 122, 206),
+      'light': rgb(180, 224, 250),
+      'lighter': rgb(235, 244, 250)
     ),
   'teal':
     (
-      'lighter': rgb(224, 245, 245),
-      'light': rgb(183, 236, 236),
-      'base': rgb(71, 193, 191),
+      'text': rgb(64, 83, 82),
+      'darker': rgb(0, 49, 54),
       'dark': rgb(0, 132, 142),
-      'darker': rgb(0, 49, 53),
-      'text': rgb(64, 83, 82)
+      'teal': rgb(71, 193, 191),
+      'light': rgb(182, 236, 235),
+      'lighter': rgb(224, 245, 244)
     ),
   'green':
     (
-      'lighter': rgb(227, 241, 223),
-      'light': rgb(187, 229, 179),
-      'base': rgb(80, 184, 60),
-      'dark': rgb(16, 128, 67),
+      'text': rgb(65, 79, 62),
       'darker': rgb(23, 54, 48),
-      'text': rgb(65, 79, 62)
+      'dark': rgb(16, 128, 67),
```

Comment:
`base` colors are missing in this file - the `getShade` function needs to be adapted for this use case.